src/devices/Nmea0183/Sentences/FluidData.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnitsNet;
+
+#pragma warning disable CS1591
```

Comment:
Why deactivating the intelissence comments?

Reply:
Because I'm lazy... No, I sometimes add this line when I create a class so that I don't have to write the documentation before the final layout of the class is set. Of course, I should have removed this...